build/images/ovs/build.sh

```diff
@@ -86,86 +97,50 @@ if [ "$DISTRO" != "ubuntu" ] && [ "$DISTRO" != "ubi" ]; then
     exit 1
 fi
 
-THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-
 pushd $THIS_DIR > /dev/null
 
 OVS_VERSION=$(head -n 1 ../deps/ovs-version)
 
 BUILD_TAG=$(../build-tag.sh)
 
-# This is a bit complicated but we make sure that we only build OVS if
-# necessary, and at the moment --cache-from does not play nicely with multistage
-# builds: we need to push the intermediate image to the registry. Note that the
-# --cache-from option will have no effect if the image doesn't exist
-# locally.
-# See https://github.com/moby/moby/issues/34715.
-
 if $PULL; then
-    if [[ ${DOCKER_REGISTRY} == "" ]]; then
-        docker pull $PLATFORM_ARG ubuntu:22.04
-    else
-        docker pull ${DOCKER_REGISTRY}/antrea/ubuntu:22.04
-        docker tag ${DOCKER_REGISTRY}/antrea/ubuntu:22.04 ubuntu:22.04
-    fi
     if [ "$DISTRO" == "ubuntu" ]; then
-        IMAGES_LIST=(
-            "antrea/openvswitch-debs:$BUILD_TAG"
-            "antrea/openvswitch:$BUILD_TAG"
-        )
-    elif [ "$DISTRO" == "ubi" ]; then
-        IMAGES_LIST=(
-            "antrea/openvswitch-rpms:$BUILD_TAG"
-            "antrea/openvswitch-ubi:$BUILD_TAG"
-        )
-    fi
-    for image in "${IMAGES_LIST[@]}"; do
         if [[ ${DOCKER_REGISTRY} == "" ]]; then
-            docker pull $PLATFORM_ARG "${image}" || true
+            docker pull $PLATFORM_ARG ubuntu:22.04
         else
-            rc=0
-            docker pull "${DOCKER_REGISTRY}/${image}" || rc=$?
-            if [[ $rc -eq 0 ]]; then
-                docker tag "${DOCKER_REGISTRY}/${image}" "${image}"
-            fi
+            docker pull ${DOCKER_REGISTRY}/antrea/ubuntu:22.04
+            docker tag ${DOCKER_REGISTRY}/antrea/ubuntu:22.04 ubuntu:22.04
         fi
-    done
+    elif [ "$DISTRO" == "ubi" ]; then
+        docker pull $PLATFORM_ARG centos:centos7
```

Comment:
Not related to this PR, is there any plan to upgrade this centos version? I found that centOS 7 will be EOL on June 2024.

Reply:
This should probably be done as part of https://github.com/antrea-io/antrea/issues/5725